### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by andreaTP**

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ entries:
     version: 0.4.2
   - apiVersion: v2
     appVersion: 0.4.1
```
Comment (patriknw) [inline index.yaml]: shall we remove all before 0.4.2 while we are at it?

File: index.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ entries:
   akka-operator:
   - apiVersion: v2
```
Comment (patriknw) [inline index.yaml:4]: what is apiVersion? is that the api of Helm or our operator?

Comment (RayRoestenburg) [reply index.yaml:4]: ~ours~ hmm. not sure

Comment (patriknw) [reply index.yaml:4]: what do we have that is v2? the crd is v1 and the image/chart version is 0.4.2

Comment (RayRoestenburg) [reply index.yaml:4]: I think it is chart api version (so helm) sorry for confusion.

Comment (RayRoestenburg) [reply index.yaml:4]: also index.yaml is generated

Comment (RayRoestenburg) [reply index.yaml:4]: @andreaTP ? in cloudflow index this is all set to v1..

Comment (andreaTP) [reply index.yaml:4]: Is generated, I'll check after lunch

Comment (andreaTP) [reply index.yaml:4]: Comes from `Chart.yaml`:
https://github.com/lightbend/akka-operator-helm/blob/master/akka-operator/Chart.yaml#L1

Should we revert it to v1 and re-publish?

Comment (andreaTP) [reply index.yaml:4]: No, it indicates that we mandate Helm3 and I guess is fair to do so:
https://helm.sh/docs/topics/charts/#the-apiversion-field

Comment (patriknw) [reply index.yaml:4]: Makes sense to correlate that to the v1 in the crd.

We can do that and release 0.4.3 (and remove 0.4.2).

Comment (patriknw) [reply index.yaml:4]: final call?
add a comment there for future generations.

Helm 3 is what we want to support.

Comment (andreaTP) [reply index.yaml:4]: done

**Comment by andreaTP** [comment]
@patriknw ready to go?

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**